docs/featurestore/featurestore.rst

```diff
@@ -843,10 +849,57 @@ Hopsworks relies on certificates being available in the Databricks cluster in or
+
+In the Databricks UI, go to *Clusters* and select your cluster. Select *Libraries*. Make sure that hopsworks-cloud-sdk is not installed (uninstall it if it is). Then *Install New*. As *Library Source* choose *PyPI* and fill in *hops-util-py* into the *Package* field.
+
+**Setting up the cluster to use hops-util-py**
+
+After installing the hops-util-py library, restart the cluster and open a Databrick notebooks connected to this cluster. Run the following command in this notebook::
+
+ import hops.featurestore as fs
+ fs.setup_databricks('my_instance', 'my_project', region_name='my_aws_region')
++------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Replace *my_instance* by the address of your instance, *my_project* by the name of your project and *my_aws_region* by the AWS region in which you stored the API Key. |
++------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+This will return two configurations that you need to add to your cluster configuration. To set them go to *Clusters*, select your cluster and click *Edit*. 
+Then, select *Configuration* and *Advanced Options*. 
+
+For the first configuration select *Init Scripts* and copy the init script path returned by *setup_databricks* in *Init Script Path* and click *Add*.
+
+For the second configuration select *Spark* and copy the spark configuration returned by *setup_databricks* in *Spark Config*.
+
+Once you have set these two configurations, click *Confirm and Restart*.
+
+**Connecting to the Feature Store**
+
+In the Databricks notebooks connected to the prepared cluster use the following code to connect to the feature store::
+
+ import hops.featurestore as fs
+ fs.connect('my_instance', 'my_project', region_name='my_aws_region')
++------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Replace *my_instance* by the address of your instance, *my_project* by the name of your project and *my_aws_region* by the AWS region in which you stored the API Key. |
++------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+.. note::
+    It is possible to connect to a project different from the one you used when calling *setup_databricks* or when last running the cluster. 
+    But, Spark is keeping open and reusing the connections to Hive and the Name Node. So you may need to restart the cluster if you want to change the project you are connected to.
```

Comment:
@berthoug what does `may` need mean? It is not deterministic?

Reply:
It happens only if they have already run an sql or hosfs request. I first tried to formulate it giving the information but it turned out more confusing than helping. I am open to suggestions.

Reply:
A suggestion is 
```
However, if you want to change the project and you have previously ran SparkSQL, connected to Hive, or read a file from HopsFS (hdfs) then it is necessary to restart the cluster. That is because Spark in the background keeps connections open to these services.